Riot/Modules/Room/Views/BubbleCells/Styles/Bubble/BubbleRoomCellLayoutUpdater.swift

```diff
@@ -180,13 +179,12 @@ class BubbleRoomCellLayoutUpdater: RoomCellLayoutUpdating {
     }
     
     private func getOutgoingMessageTextViewMargins(from bubbleCell: MXKRoomBubbleTableViewCell) -> UIEdgeInsets {
-            
-        let innerContentLeftMargin: CGFloat = 57
         
         let messageViewMarginTop: CGFloat = 0
         let messageViewMarginBottom: CGFloat = 0
-        let messageViewMarginLeft: CGFloat = 80.0 + innerContentLeftMargin
-        let messageViewMarginRight: CGFloat = 78.0
+        let messageViewMarginLeft =
+        BubbleRoomCellLayoutConstants.outgoingBubbleBackgroundMargins.left + RoomBubbleCellLayout.innerContentViewMargins.left
```

Comment:
Just a double check that this is adding `innerContentViewMargins` whereas the right margins are adding `bubbleTextViewInsets`.

Reply:
Good catch